scheduler/src/cook/scheduler/scheduler.clj

```diff
@@ -686,13 +686,12 @@
         (reduce into [] task-txns)
         (fn [e]
           (log/warn e
-                    "Transaction timed out, so these tasks might be present"
+                    "In" pool-name "pool, transaction timed out, so these tasks might be present"
                     "in Datomic without actually having been launched in Mesos"
                     matches)
           (throw e))))
-    (log/info "Launching" (count task-txns) "tasks")
+    (log/info "In" pool-name "pool, launching" (count task-txns) "tasks")
     (ratelimit/spend! ratelimit/global-job-launch-rate-limiter ratelimit/global-job-launch-rate-limiter-key (count task-txns))
-    (log/debug "Matched tasks" task-txns)
```

Comment:
why don't you want this logged?

Reply:
Because it's already being logged before this:

https://github.com/twosigma/Cook/pull/1322/files#diff-6c16cf7360b42776defb5ddcfa4d6f95R677

Reply:
Not sure if that link works well, but it's being logged with this line:

```
(log/info "In" pool-name "pool, writing tasks" task-txns)
```

on line 677 of the patched file.